cpp/src/text/edit_distance.cu

```diff
@@ -143,7 +292,7 @@ std::unique_ptr<cudf::column> edit_distance(cudf::strings_column_view const& inp
                     thrust::counting_iterator<cudf::size_type>(0),
                     thrust::counting_iterator<cudf::size_type>(input.size()),
                     offsets.begin(),
-                    calculate_compute_buffer_fn{*d_strings, *d_targets});
+                    calculate_compute_buffer_fn{*d_strings, *d_targets, 2, 3});
```

Comment:
Can we use named constants here? If I'm understanding the approach correctly, the count is 3 because we're storing 3 diagonals at a time. I don't fully grasp why the padding is 2 though. Processing the Levenshtein distance for two strings of length M and N requires a matrix of size `min(M, N) + 1`, and the longest diagonal is also that size, so why isn't the padding 1?